### ISSUE TITLE
comments: dropdown not usable without mouse

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#4046]
**URL:** https://meinberlin-dev.liqd.net/ideas/2021-00640/
**user:** any without mouse
**expected behaviour:** can use drop down to change/delete comments or replys without my mouse
**behaviour:** cannot chose from drowdown
**important screensize:**
**device & browser:** tested big sur chrome and edge via browserstack
**Comment/Question:** is that an a4 issue?

Screenshot
<img width="301" alt="Bildschirmfoto 2021-12-08 um 14 01 59" src="https://user-images.githubusercontent.com/35491681/145212791-05adc1c0-6881-466e-8f7d-26bdec2887d2.png">

**Comment by phillimorland** [comment]
as we will change the comments anyway I will add a don't fix onto this

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland totally right, will close the issue then!
